core/modules/layout/includes/layout.layout.inc

```diff
@@ -85,13 +85,18 @@ function layout_layout_access_info() {
   // Provide type-specific plugins for each entity type.
   foreach ($entity_types as $entity_type => $entity_info) {
 
+    $entity_label = $entity_info['label'];
+    if ($entity_label == 'Node') {
+      $entity_label = 'Content';
```

Comment:
```suggestion
      $entity_label = t('Content');
```

...and not 100% sure if the translated labels are being stored for for entities in multilingual/translated site. If so, then the conditional should also be `== t('Node')` instead.

Reply:
...actually this might be config translation territory here.

Reply:
These are just values in the info at this point, so they shouldn't be translated yet. They'll be translated when they are rendered.